definitions/npm/ramda_v0.x.x/flow_v0.82.x-/ramda_v0.x.x.js

```diff
@@ -1560,19 +1560,90 @@ declare module ramda {
   ): (key: T) => $ElementType<O, T>;
   declare function prop<T: string, O>(key: T, o: O): $ElementType<O, T>;
 
-  declare function propOr<T, V, A: { [k: string]: V }>(
+  // propOr 1 argument => (function (2 arguments) or (function (1 argument) => function (1 argument)))
+  declare function propOr<T, V: string, A: Object, J: (string | number | null | void)>(
```

Comment:
Can you add a legend for what T, V, A, and J stand for? It'll make it easier to track.